spec/features/deposit_via_xlsx_spec.rb

```diff
@@ -1,13 +1,31 @@
 # frozen_string_literal: true
 
+require 'rubyXL'
+require 'rubyXL/convenience_methods/cell'
+
 RSpec.describe 'Use Argo to upload metadata in a spreadsheet', type: :feature do
-  let(:start_url) { 'https://argo-stage.stanford.edu/view/druid:qc410yz8746' }
+  let(:argo_home) { 'https://argo-stage.stanford.edu/' }
+  let(:start_url) { "#{argo_home}view/druid:qc410yz8746" }
+  let(:druid1_url) { "#{argo_home}view/fh141gk9610" }
+  let(:druid2_url) { "#{argo_home}view/mw605wr9855" }
+  let(:spec_location) { 'spec/fixtures/filled_template.xlsx' }
+  let(:title1) { RandomWord.phrases.next }
+  let(:title2) { RandomWord.phrases.next }
   let(:note) { RandomWord.phrases.next }
 
   before do
+    filled_xlsx = RubyXL::Parser.parse(spec_location)
+    sheet_one = filled_xlsx.worksheets[0]
+    sheet_one[2][3].change_contents title1
+    sheet_one[3][3].change_contents title2
+    filled_xlsx.write(spec_location)
```

Comment:
how about writing to a new location rather than overwriting? Then we can cleanup just by removing the tempfile and we don't have to call out to git?

Reply:
Done, now writing to temp file.